ovn-tester/ovn_tester.py

```diff
@@ -118,7 +128,23 @@ def read_config(configuration):
         bringup_cfg = ClusterBringupCfg(
             n_pods_per_node=bringup_args.get('n_pods_per_node', 10)
         )
-        return log_cmds, cluster_cfg, brex_cfg, bringup_cfg
+
+        netpol_multitenant_args = config.get('netpol_multitenant', dict())
+        ranges=[
+            NsRange(
+                start=range_args.get('start', 0),
+                n_pods=range_args.get('n_pods', 5),
+            ) for range_args in netpol_multitenant_args.get('ranges', list())
+        ]
+        ranges.sort(key=operator.itemgetter(0), reverse=True)
```

Comment:
This might be a bit more readable as:
```
ranges.sort(key=operator.attrgetter('start'), reverse=True)
```
This at least makes it more clear that the `start` attribute is the key.
Or if you want to avoid importing `operator`, then you could go with:
```
ranges.sort(key=lambda x: x.start, reverse=True)
```

Reply:
I went for `ranges.sort(key=lambda x: x.start, reverse=True)`, thanks for the suggestion.